### PR TITLE
Add spec for ls_ocp_cni_openshift_sdn

### PR DESCRIPTION
**Pull request by shzhou12**

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -388,6 +388,11 @@
             "pattern": [],
             "symbolic_name": "ls_var_spool_clientmq"
         },
+        {
+             "command": "/bin/ls -l /var/lib/cni/networks/openshift-sdn",
+             "pattern": [],
+             "symbolic_name": "ls_var_lib_cni_networks_openshift-sdn"
```
Comment (bfahr) [inline uploader.json]: @shzhou12 this symbolic name needs to match the name of the spec in insights core `ls_ocp_cni_openshift_sdn`.

File: uploader.v2.json [hunk 2]
```diff
@@ -388,6 +388,11 @@
             "pattern": [],
             "symbolic_name": "ls_var_spool_clientmq"
         },
+        {
+             "command": "/bin/ls -l /var/lib/cni/networks/openshift-sdn",
+             "pattern": [],
+             "symbolic_name": "ls_var_lib_cni_networks_openshift-sdn"
```
Comment (bfahr) [inline uploader.v2.json]: Same comment as above.

Comment (shzhou12) [reply uploader.v2.json]: @bfahr Thanks for your review. I have changed the code according to your comments. Could you please help to review the changes?

**Comment by YakovLittle** [comment]
test this please

**Comment by bfahr** [comment]
This PR is approved.

**Pull request merged**